CMakeLists.txt

```diff
@@ -583,7 +580,7 @@ psibase_package(
     PACKAGE_DEPENDS NftSys ProxySys AccountSys
     SERVICE supervisor-sys
         DATA ${CMAKE_CURRENT_SOURCE_DIR}/services/user/SupervisorSys/ui/dist /
-    DEPENDS SupervisorSys_js ${CMAKE_CURRENT_SOURCE_DIR}/services/user/SupervisorSys/ui/dist/index.html
+    DEPENDS ${SupervisorSys_js_DEP} ${CMAKE_CURRENT_SOURCE_DIR}/services/user/SupervisorSys/ui/dist/index.html
```

Comment:
`.../dist/index.html` is included in `${SupervisorSys_js_DEP}`, which is the reason I suggested the replacement.

Reply:
Fixed